parent/pom.xml

```diff
@@ -197,6 +197,12 @@
         <version>${project.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>com.google.code.findbugs</groupId>
+        <artifactId>jsr305</artifactId>
+        <version>3.0.2</version>
```

Comment:
where is this used ?

Reply:
It is used for Nullable annotation here : https://github.com/fabric8io/fabric8-maven-plugin/blob/7d7b13e3f675543d09e56eccc71f20abd42be0ae/enricher/fabric8/src/main/java/io/fabric8/maven/enricher/fabric8/VertxHealthCheckEnricher.java#L66